OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -52,12 +52,25 @@ public sealed class HackyAIInfo : IBotInfo, ITraitInfo
 		[Desc("Minimum excess power the AI should try to maintain.")]
 		public readonly int MinimumExcessPower = 0;
 
-		[Desc("How long to wait (in ticks) between structure production checks when there is no active production.")]
+		[Desc("Minimum delay (in ticks) between structure production checks when there is no active production.")]
 		public readonly int StructureProductionInactiveDelay = 125;
 
-		[Desc("How long to wait (in ticks) between structure production checks ticks when actively building things.")]
+		[Desc("Minimum delay (in ticks) between structure production checks when actively building things.")]
 		public readonly int StructureProductionActiveDelay = 10;
 
+		[Desc("A random delay (in ticks) of up to this is added to production delays.")]
+		public readonly int StructureProductionRandomBonusDelay = 10;
+
+		[Desc("How long to wait (in ticks) until retrying to build structure after the last 3 consecutive attempts failed.")]
```

Comment:
Yes, and this does not add a random bonus delay, so it represents the exact delay value. The two Active and Inactive delays above add a random bonus delay, though, so they only represent the minimum delay. So the descriptions are correct, though it seems they can be misunderstood, so I'll try to word them better.